parlai/crowdsourcing/tasks/acute_eval/README.md

```diff
@@ -83,12 +87,131 @@ In our paper, we address the problem of wording the questions and binary choices
+
+- Specifying the minimum dialogue length to be counted as valid for analysis
+- Specifying the maximum number of matchups per model pair visualized in HTML
+- Optionally including a checkbox column for annotating the convo pairs in HTML
+
+If you would like to make use of these old features, please open a [ParlAI issue](https://github.com/facebookresearch/ParlAI/issues) or restore the original version by switching to the `acute_eval` tag of ParlAI:
+
+```bash
+$ git checkout acute_eval
+```
+
+## Setup
+
+### 1. Determine what you will be evaluating and create a config file
+
+This is an important step - do you have conversation logs between a model and a human? Would you like to evaluate model self-chat? Do you want to evaluate dataset logs?
+
+Each of these options involves _slightly_ different preparation. However, each involves specifying a config file, which is specified on the command line by `mephisto.blueprint.config_path` when launching Fast ACUTEs.
+
+In the `task_config/` folder, you will find several `model_config_*.json` files that map a _unique_ identifier to appropriate configuration arguments; these arguments differ depending on what you will be evaluating.
+
+A few of these options are enumerated below.
+
+#### Model self-chat
+
+If you would like to evaluate a model chatting to itself, you simply specify the appropriate model parameters in the config. The parameters are any that you would need to specify on the command line, and include things like the model-file, fixed candidates file, etc. You can see an example in `task_config/model_config_self_chat.json`.
+
+#### JSONL logs
+
+If you have logs in the appropriate JSONL format, as would be generated by the self-chat script, then all you need to specify is the `log_path` and whether the logs are model self-chats. You can see an example in `task_config/model_config_logs.json`.
+
+The appropriate JSONL format is one that can be read by ParlAI's [Conversations](https://github.com/facebookresearch/ParlAI/blob/master/parlai/utils/conversations.py) class. Note that the identifier in the config should match **EXACTLY** the `id` of the model in the conversations.
+
+#### Dataset
+
+If you'd like to evaluate examples from a dataset available in ParlAI directly, simply specify the `task` in the config. You can see an example in `task_config/model_config_dataset.json`.
+
+### 1b. (Optional) Determine the self-chat task that you will use
+
+If you will be evaluating models via self-chat, you will need to determine the ParlAI task you will use to help generate the self-chats. This is not so much any work on your part other than identifying a task that is set up for self-chat, i.e., a task that has the appropriate worlds for conducting self-chat with the models. Optionally, you may want to introduce context, e.g. as in `convai2` or `blended_skill_talk`: this can be specified in the config file with the `'prepended_context'` parameter.
```

Comment:
This should likely just be a subpoint of the self-chat option, rather than placed here.

Reply:
Good point, thanks!